Tools/Scripts/compare-webkit-configurations

```diff
@@ -60,6 +60,8 @@ def flag_for_comparison(comparison_name):
         return "--remote-layer-tree"
     if comparison_name == "accessibility-isolated-tree":
         return "--accessibility-isolated-tree"
+    if comparison_name == "site-isolation"
+        return "--site-isolation"
```

Comment:
I think this change (along with the others from `Tools/Scripts`) should be in a different PR which is basically `Add --site-isolation to run-webkit-tests`, which should land first.